quota/quota.go

```diff
@@ -234,12 +234,6 @@ func ResolveCrossReferences(grps map[string]*Group) error {
 			}
 			grp.parentGroup = parent
 
-			// also add an internal link from the parent to this group, this
```

Comment:
thanks for spotting that this was unnecessary, this was part of a previous implementation about doing the cross-checks that parents specify children and vice-versa